boards/arm64/phycore_am62x_a53/doc/index.rst

```diff
@@ -0,0 +1,117 @@
+.. _phycore_am62x_a53:
+
+PHYTEC phyCORE-AM62x (Cortex-A53)
+#################################
+
+Overview
+********
+
+PHYTEC phyCORE-AM62x is board is based on TI Sitara applications
+processor, composed of a quad Cortex®-A53 cluster and a single Cortex®-M4 core.
+Zephyr OS is ported to run on the Cortex®-A53 core.
+
+- Board features:
+
+  - RAM: 2GB DDR4
+  - Storage:
+
+    - 16GB eMMC
+    - 64MB OSPI NOR
+    - 4KB EEPROM
+  - Ethernet
+
+More information about the board can be found at the
+`PHYTEC website`_.
+
+Supported Features
+==================
+
+The Zephyr phycore_am62x_a53 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| GIC-v3    | on-chip    | interrupt controller                |
++-----------+------------+-------------------------------------+
+| ARM TIMER | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 200 MHz.
+
+DDR RAM
+-------
+
+The board has 2GB of DDR RAM available. This board configuration
+allocates Zephyr 1MB of RAM (0x82000000 to 0x82100000).
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+CPU's UART0.
+
+U-Boot
+******
+
+The factory SD card from PHYTEC contains a U-Boot image that does not have
+the cache commands enabled. These commands are required for a successful
+boot of Zephyr. Follow these steps to rebuild U-Boot.
+
+Install U-Boot dependencies:
+https://u-boot.readthedocs.io/en/latest/build/gcc.html#dependencies
+
+Clone the PHYTEC TI fork of U-Boot:
```

Comment:
Is this really needed? Can you check with upstream UBoot once? AM62 has fairly good upstream uboot support.

Reply:
Only these PHYTEC boards are supported in upstream U-Boot:

- phycore-am335x-r2-regor_defconfig
- phycore-am335x-r2-wega_defconfig
- phycore-imx8mm_defconfig
- phycore-imx8mp_defconfig
- phycore-rk3288_defconfig
- phycore_pcl063_defconfig
- phycore_pcl063_ull_defconfig 

But feel free to add TI AM62X EVM boards after this is merged

Reply:
@gramsay0 - Thanks for this PR! We will send our u-boot upstream soon because we internally focus on the kernel first.

I will send a patch and add the cache cmd to our u-boot.

Reply:
@dnltz thanks! We can update the docs once that's merged.
`CONFIG_CMD_MP` would also be useful for testing SMP